### ISSUE TITLE
New Emote on join menu code should be merged with the wheel

### ISSUE THREAD

**Issue by Jupeyy** [ddnet/ddnet#5057]
I like this feature, but I don't understand why we need a new menu entry for it:

If I want an emote from the emote wheel, i purposly click it anyway, so just save it here.
If i missclicked I gonna repick a new one anyway

I don't understand what problem the new menu solves.

@sjrc6 maybe you can explain it to me, maybe I'm just stupid rn :D

**Comment by sjrc6** [comment]
I like that the menu displays all the eyes on your current skin, I also think it makes it easier to manage eyes for dummy. Selecting eye from wheel ingame seems like it has pretty annoying rate limits if you change more than a few times. 

I don't have any issue with the menu component being removed if others agree I just wanted to present it as an option. Personally I like it.

**Comment by C0D3D3V** [comment]
> I like that the menu displays all the eyes on your current skin,

Do you not already see all eyes in the wheel?
![grafik](https://user-images.githubusercontent.com/14315968/166152672-0d03cdbb-dbd4-4354-a3e6-f178a64ba779.png)

**Comment by heinrich5991** [comment]
> Do you not already see all eyes in the wheel?

Not in the settings, where you select skins.

**Comment by Jupeyy** [comment]
> I like that the menu displays all the eyes on your current skin, I also think it makes it easier to manage eyes for dummy.

Yes that's a good argument. Now i wonder if the skinlist should also show the current selected emote ^^

**Comment by sjrc6** [comment]
If we are going to keep the settings menu should we still have the emote wheel set the default emote? I didn't do it initially because it technically changes the behavior of the wheel but maybe it would be better that way.

**Comment by Jupeyy** [comment]
I think it would be useful, i'd much more use the wheel then switching to a menu for such a simple thing.

Other opinions?

**Comment by C0D3D3V** [comment]
Is it now about the question whether the emote wheel can also be used to set the default emote? Yes, I would find that good. If a player changes the server, the last emote may be used as initial emotion that he set via emote wheel. 

On the other hand, you have the option for the standard emote in the settings. As a user, you would expect that the emote is reset to the default emote when you disconnect. So I would probably not set the emote in the emote wheel as default emote, because it would be less intuitive.

I would like it if all skins would adopt the default emote you choose in the settings. Since this setting is most likely a personal choice of what type of emote you like best, this choice should also affect all other skins.

**Comment by Jupeyy** [comment]
> On the other hand, you have the option for the standard emote in the settings. As a user, you would expect that the emote is reset to the default emote when you disconnect. 

From the status quo point of view maybe.
Generally I am used to my settings being saved in most games (be it sorting of inventory, making your helmet invisible or whatever)

**Comment by Jupeyy** [comment]
Btw independent of the wheel discussion. I agree it would be nice to have menu tees also adopt the emotes
good little nice to have feature
